ci/teams/rhtap-servicerelease/release-external-secrets.yaml

```diff
@@ -9,4 +9,4 @@ spec:
   application: external-secrets
   displayName: ''
   origin: rhtap-gitops-tenant
-  releaseStrategy: push-to-external-registry-strategy
+  releaseStrategy: rhtap-service-push-strategy
```

Comment:
Is the new strategy a superset of the old one?

Reply:
yes! adds the PR creation and optional slack notification